src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -151,7 +151,7 @@ public void truncate() throws ConnectException {
 
   @Override
   public void close() throws ConnectException {
```

Comment:
is there any where in the plugin code that this close() is supposed to be called? would it be useful to also add some logging there?

Reply:
It's called by close in the TPW (https://github.com/confluentinc/kafka-connect-hdfs/blob/master/src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java)

Reply:
What logging did you have in mind?

Reply:
as long as you think there are sufficient log leading up to this part then that should be fine. I guess my only worry was, it's possible the code somehow never reached to this point, then this line won't show up at all and we'd still need some info to help us trace back to where we think should be invoked. Maybe DEBUG level should be giving us all those info?

Reply:
gotcha, yes I will add some on higher level